pkg/config/parse_test.go

```diff
@@ -284,59 +284,6 @@ func TestParseExampleConfig(t *testing.T) {
 	restoreEnv(envVarBackup)
 }
 
-// TestConfigOverridesDefault validates that a value provided by the config is not overwritten during parsing
```

Comment:
I removed this test because it's inaccurate: we test that env vars don't get overwritten. This test fails if you actually do have an env var for minio set up...so it's unstable and inaccurate. We should just test that config works as intended: toml gets parsed, and env vars override toml. Which we do in the rest of this file.